src/applications/check-in/travel-claim/pages/complete/index.jsx

```diff
@@ -1,19 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useTranslation } from 'react-i18next';
 
-import Wrapper from '../../../components/layout/Wrapper';
+import TravelCompleteDisplay from '../../../components/pages/TravelComplete/TravelCompleteDisplay';
 
 const Complete = () => {
-  const { t } = useTranslation();
-
-  return (
-    <Wrapper pageTitle={t('fpo-header')} classNames="travel-page">
-      <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-align-itmes--stretch small-screen:vads-u-flex-direction--row">
-        Complete!
-      </div>
-    </Wrapper>
-  );
+  return <TravelCompleteDisplay />;
```

Comment:
I don't feel like we need to have shared display components for these unique pages. Not a huge deal, but it feels like extra overhead. Especially since these wrappers don't have a unit test, counting against our coverage. I feel like if we keep this pattern, we should add unit tests as pointless as they may seem.

Reply:
I agree and moved everything to the app folder